pkg/cluster/cluster.go

```diff
@@ -849,8 +849,8 @@ func (c *Cluster) GetCurrentProcess() Process {
 }
 
 // GetStatus provides status of the cluster
-func (c *Cluster) GetStatus() *ClusterStatus {
-	return &ClusterStatus{
+func (c *Cluster) GetStatus() *Status {
```

Comment:
What exactly golint was saying about this? If it's just a renaming suggestion, I would recommend to revert it, because there are WorkerStatus, ClusterStatus, PostgresStatus etc. and I think it's better to distinguish all of them.

Reply:
it makes stutter warning because func name is including package name ‘cluster’ 

ref: https://github.com/giantswarm/inago/issues/38

Reply:
Is this ever a problem? Do we have multiple status objects of different kind in same places? How does Go resolve this?